src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -231,6 +233,15 @@ public StreamingResponseBody streamEvents(
         };
     }
 
+    /**
+     * Every consumer identifies itself using a client-id to use its quota. The client id is a combination of
+     * application name and event type so that every application can consume up to the quota limit per partition.
+     * Leader partitions from a single event type are guaranteed to be located on different brokers.
```

Comment:
Is that always true? If some brokers are down, there could be less brokers than the number of partitions in a single event type, and this would not hold true anymore. Of course it would be temporary, until new brokers are up.